fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CatalogAnalyzer.java

```diff
@@ -125,8 +140,15 @@ public Void visitAlterCatalogStatement(AlterCatalogStmt statement, ConnectContex
                         (ModifyTablePropertiesClause) statement.getAlterClause();
                 Map<String, String> properties = modifyTablePropertiesClause.getProperties();
 
+                String catalogType = GlobalStateMgr.getCurrentState().getCatalogMgr().getCatalogType(statement.getCatalogName());
                 for (Map.Entry<String, String> property : properties.entrySet()) {
-                    if (!property.getKey().equals("ranger.plugin.hive.service.name")) {
+                    String confName = property.getKey();
+
+                    if (("hive".equals(catalogType) || "hudi".equals(catalogType))) {
+                        if (!SUPPORT_ALTER_PROPERTIES.contains(confName)) {
```

Comment:
why only support partial properties？ all valid proprties should support because alredy recreate the catalog

Reply:
It is hard to determine which property provided by the user are legitimate.

Reply:
at create connector phase, each connector could check whether property value  is legitimate, so there is no need to check